console/src/telemetry/Actions.ts

```diff
@@ -38,46 +47,46 @@ const setConsoleOptsInDB = (opts, successCb, errorCb) => (
     ),
   };
 
-  return dispatch(requestAction(url, options)).then(
-    data => {
+  return (dispatch(requestAction(url, options)) as any).then(
+    (data: object) => {
       if (successCb) {
         successCb(data);
       }
     },
-    error => {
+    (error: Error) => {
       if (errorCb) {
         errorCb(error);
       }
     }
   );
 };
 
-const telemetryNotificationShown = () => dispatch => {
+const telemetryNotificationShown = () => (dispatch: (arg: Partial<SetNotificationShowAction>) => void) => {
   dispatch({ type: SET_NOTIFICATION_SHOWN });
 };
 
 const setTelemetryNotificationShownInDB = () => {
-  const successCb = data => {
+  const successCb: SuccessCb = (data: object) => {
```

Comment:
No need, and we can remove `SuccessCb` type and inline it.
```suggestion
  const successCb = (data: object) => {
```